### ISSUE TITLE
Output channel keeps stealing focus when ie. WF server writes to the output

### ISSUE THREAD

**Issue by odockal** [redhat-developer/vscode-rsp-ui#156]
Link: https://github.com/redhat-developer/vscode-server-connector/issues/473

**Comment by robstryker** [comment]
Closing since there is already a setting to override this behavior.  See redhat-developer/vscode-server-connector#473

If you go to `File -> Preferences -> Settings` and search for `servers` you should see this setting:

```
Vscode Adapters: Show Channel On Server Output
Show Server's output channel when new text added to output stream.
```

You should be able to uncheck this setting.

**Comment by gaspo100** [comment]
Not sure if this is the same issue or a new one but the focus stealing has not been completely resolved in the latest VS Code (1.70.2) and the Community Server Connector (0.23.13).

I have the "showChannelOnServerOutput" unchecked, the Output window does not focus but instead the Servers panel now focuses on itself whenever I save the file or perform other actions in the VS Code UI.
![image](https://user-images.githubusercontent.com/931540/187140839-d648b790-38a1-4b4a-a3ec-8a25f040c729.png)
![image](https://user-images.githubusercontent.com/931540/187141061-d8177861-169f-4c71-984d-04773745b819.png)
